<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoDisplayClass</class>
 <widget class="QDialog" name="VideoDisplayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>视频浏览器</string>
  </property>
  <widget class="QPushButton" name="receiveButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>接收视频</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>520</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="displayLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>displayLabel</zorder>
  <zorder>receiveButton</zorder>
  <zorder>exitButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="videodisplay.qrc"/>
 </resources>
 <connections/>
</ui>
